Areas/Identity/Data/WhaleSpottingDbContext.cs

```diff
@@ -0,0 +1,26 @@
+using System;
```

Comment:
In my mind this was part of the last pull request, and either this wasn't needed or something similar has already been merged

Reply:
I scaffolded the app again yesterday to get a copy of the logout files. So we gained some extra files again. The ValidationScriptsPartial was generated alongside the the logout files.  :)

Reply:
For this file in particular it is exactly the same as the old one, think it's been deleted and regenerated upon scaffolding again, that's why it appears to be 'new' :)